org.gnome.PasswordSafe.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id" : "org.gnome.PasswordSafe",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.30",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "gnome-passwordsafe",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+        "--filesystem=/usr/share/icons:ro",
```

Comment:
Why?

Reply:
Access to /usr/share/icons can be removed, its an ancient relic.
Network access is needed when copying the stock database into a gvfs-goa drive. If this permission isn't set this will not work.

Reply:
Removed the filesystem permission to /usr/share/icons